### PR TITLE
Introduce react-router

### LINKED ISSUE #17: React Router groundwork
We are going to use React Router to simulate different 'pages' in our app. Lay the groundwork for using React Router, including any documentation that may be necessary for other people to be able to use it.

We are not going to use server-side rendering, so this should be easier than usual.

Feel free to consult and collaborate with @duci9y on this.

### PR DESCRIPTION
**Pull request by controversial**
This PR adds a basic `react-router` setup with a couple of routes. The routes are not intended to necessarily have the same URL pattern as the final app, but to serve as an example for how routing might work for a few routes of an app like TUTV.

Closes #17

### CODE REVIEW

**Comment by controversial** [comment]
Documentation completed at https://github.com/JumboCode/TUTV/wiki/Routing-with-react-router. This documentation reflects the current state of the PR, including decisions about file structure etc, which are not necessarily intended to be final. I'll update the docs if anything changes here.

File: src/frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const Home: React.FC = () => {
+  return (
+    <div>
+      <h1>Home page</h1>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (duci9y) [inline src/frontend/src/pages/Home.tsx]: Maybe this should also be in its own folder to ensure consistency? Thoughts?

**Comment by controversial** [comment]
Yeah, let’s do that. I’ll move it

**Comment by controversial** [comment]
I moved the Home page component into its own folder

**Comment by controversial** [comment]
Travis seems like it's having issues

**Pull request merged**